SQL/SQL.sublime-syntax

```diff
@@ -8,10 +8,7 @@ file_extensions:
   - ddl
   - dml
 
-first_line_match: |-
-  (?xi:
-    ^ \s* -- .*? -\*- .*? \bsql\b .*? -\*-  # editorconfig
-  )
+first_line_match: (?xi) ^ \s* -- .*? -\*- .*? \bsql\b .*? -\*-  # editorconfig
```

Comment:
arguably, keeping this on multiple lines is better for consistency with other syntaxes https://github.com/sublimehq/Packages/commit/3629e8bce1d25fbbc66528ed212f94597cbb8800